src/pages/docs/multiple-forms.mdx

```diff
@@ -1,29 +1,34 @@
 ---
-name: Multiple Forms 
+name: Multiple Forms
 route: /docs/multiple-forms
+parent: Tutorials
 ---
 
-import { LinkedForms, MultipleForms } from './components/multiple-forms/FormsExamples'
+import {
+  LinkedForms,
+  MultipleForms
+} from './components/multiple-forms/FormsExamples';
 
 # Multiple Forms
 
-There are use cases where multiple forms should be embedded into a single page. This section 
+There are use cases where multiple forms should be embedded into a single page. This section
 describes some of these in more detail.
 
 ## Linking forms
 
 Forms can interact with each other by using `$ref` and setting the `scope` property within the UI schema
 accordingly. In this example, we have two entities, Person and Address. Whenever we change
-the `shippingAddress` property of the person, it is accordingly updated in the address form and vice versa.
+the `shippingAddress` property of the person, it is subsequently updated in the address form and vice versa.
 
 <LinkedForms />
 
-## Multiple forms within single store
-There might be use cases where you have forms that do not have anything in common. In such cases
-you can explicitly pass  `schema` and `uischema` props to the `JsonForms` component
+## Independent forms
+
+There might be use cases where you have forms that do not have anything in common, so your forms are independent, although they share the same store. In such cases
+you can explicitly pass `schema` and `uischema` props to the `JsonFormsDispatch` component
 (if you don't specify these, they will be fetched from the store).
 
-Additionally, the `JsonForms` component also takes a `path` prop which specifies
+Additionally, the `JsonFormsDispatch` component also takes a `path` prop which specifies
```

Comment:
`JsonForms` is the right one..

Reply:
Done, reverted the change.

Reply:
I think this one is still outdated.

Reply:
Actually, I think this whole paragraph is not true anymore, so we might get rid of it altogether.

Reply:
I removed the paragraph.